cluster-controller/src/main/java/io/strimzi/controller/cluster/ClusterController.java

```diff
@@ -201,13 +215,18 @@ private void recreateConfigMapWatch() {
     /**
       Periodical reconciliation (in case we lost some event)
      */
-    private void reconcile() {
```

Comment:
We should rename this method `reconcileAllClusters()` or something, because there are too many `reconcile()` methods currently.